app/__init__.py

```diff
@@ -172,6 +177,22 @@ def post_time_line_post():
     name = request.form.get("name")
     email = request.form.get("email")
     content = request.form.get("content")
+
+    email_regex = r'\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b'
+
+    # Check that fields are not empty
+    if not name or name == "":
```

Comment:
Look into Flask JSON Schema decorator to validate you received mandatory fields. @mannynacc